<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MazeGame</class>
 <widget class="QWidget" name="MazeGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>738</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>626</width>
    <height>738</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>626</width>
    <height>738</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="image_11">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>images/square1_1.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="image_32">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>images/open3_2.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="image_23">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>images/square2_3.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="image_21">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>images/open2_1.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="image_12">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>images/square1_2.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="image_31">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>images/square3_1.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="image_13">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>images/square1_3.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="image_22">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>images/open2_2.jpg</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="image_33">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>images/square3_3.jpg</pixmap>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="button_down">
         <property name="text">
          <string>Down (q)</string>
         </property>
         <property name="shortcut">
          <string>Q</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="button_right">
         <property name="text">
          <string>Right (d)</string>
         </property>
         <property name="shortcut">
          <string>D</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="button_left">
         <property name="text">
          <string>Left (a)</string>
         </property>
         <property name="shortcut">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="button_backwards">
         <property name="text">
          <string>Backwards (s)</string>
         </property>
         <property name="shortcut">
          <string>S</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="button_up">
         <property name="text">
          <string>Up (e)</string>
         </property>
         <property name="shortcut">
          <string>E</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="button_forwards">
         <property name="text">
          <string>Forwards (w)</string>
         </property>
         <property name="shortcut">
          <string>W</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="button_turn_left">
         <property name="text">
          <string>Turn Left (z)</string>
         </property>
         <property name="shortcut">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="button_turn_right">
         <property name="text">
          <string>Turn Right (c)</string>
         </property>
         <property name="shortcut">
          <string>C</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_steps">
         <property name="font">
          <font>
           <family>Georgia</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Steps: 0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_coordinates">
         <property name="font">
          <font>
           <family>Georgia</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>(X,Y,Z): (0, 0, 0)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_direction">
         <property name="font">
          <font>
           <family>Georgia</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Direction: North</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>button_down</tabstop>
  <tabstop>button_forwards</tabstop>
  <tabstop>button_up</tabstop>
  <tabstop>button_turn_left</tabstop>
  <tabstop>button_left</tabstop>
  <tabstop>button_backwards</tabstop>
  <tabstop>button_right</tabstop>
  <tabstop>button_turn_right</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
